### ISSUE TITLE
feat(http executor): Deal with binary response

### ISSUE THREAD

**Issue by lmolas** [ovh/venom#195]
I would like to have a base64 body in the Result struct in order to facilitate use of binary responses.

**Comment by nqb** [comment]
+1

**Comment by yesnault** [comment]
available in next release > 0.28.0

```
name: HTTP testsuite
vars:
  input: "this my input"

testcases:
- name: get http testcase
  steps:
  - type: http
    method: GET
    url: https://eu.api.ovh.com/1.0/
    info: foo {{.result.bodyjson | b64enc}}
    assertions:
    - result.body ShouldContainSubstring /dedicated/server
```
